### ISSUE TITLE
Build/Delete tool locking up

### ISSUE THREAD

**Issue by Garrett-C** [tsunamayo/Skywanderers#1181]
### Short Description of the issue:
Occasionally the build tool/delete tool seems to lock up. This results in it trying to build dragged shapes when you have not yet clicked. It is possible that it was caused by turning symmetry off and on again but I am unsure. 

### Environment
- Skywanderers version: Latest Build
- Graphics Card: GTX 1060

### Additional information:
![image](https://user-images.githubusercontent.com/43644633/58754378-84ec9900-84c6-11e9-9cc6-9c0fd566cf5c.png)
[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3244269/output_log.txt)

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\TsunamayoCorp\Skywanderers\`

**Comment by ZachZent** [comment]
I just got this as well. I selected the repair tool and everything froze up. When I exited a reloaded, the game screen was black, the repair tool was still selected, and the hotbar empty. 
https://cdn.discordapp.com/attachments/271723356795961345/584538210416328725/unknown.png

[28039B26-7B7D.zip](https://github.com/tsunamayo/Skywanderers/files/3244331/28039B26-7B7D.zip)
Containing the world/character in question
